Project.toml

```
@@ -29,19 +29,20 @@ G4Vis = ["Makie", "Colors", "StaticArrays", "Rotations", "LinearAlgebra", "IGLWr
 
 [compat]
 CSV = "0.10"
+Colors = "0.13"
```

if this is only imported for `colormap`, this could go as far back as 0.12.9, where this was changed the last time. 

https://github.com/JuliaGraphics/Colors.jl/commit/6bb0d5cca0524760f9a73cd7d225a8c0466e8866#diff-30b2eed98cb4313194584e592aa0fc6a51e8c537d2ed8c72ff541239dc59f90f

Probably, this could even be dated further back.

```suggestion
Colors = "0.12.9, 0.13"
```